scripts/001-binutils.sh

```diff
@@ -44,8 +44,9 @@ if [ "$(uname -s)" = "Darwin" ]; then
     TARG_XTRA_OPTS="--with-system-zlib --with-gmp=$(brew --prefix gmp) --with-mpfr=$(brew --prefix mpfr)"
   elif command -v port &> /dev/null; then
   ## Check if using MacPorts
-    MACPORT_BASE=$(dirname `port -q contents gmp|grep gmp.h`|sed s#/include##g)
-    echo Macport base is $MACPORT_BASE
+    MACPORT_BASE=$(dirname $(port -q contents gmp|grep gmp.h)|sed s#/include##g)
+    printf 'Macport base is %s\n' "$MACPORT_BASE"
+    alias sed='gsed'
```

Comment:
Is this really needed?
Unless it is necessary for `ps2toolchain-ee`, the IOP one compiled successfully without that. I know that for `brew` it was needed